flintrock/ec2.py

```diff
@@ -81,6 +81,7 @@ def instances(self):
             return self.slave_instances
 
     @property
+    @functools.lru_cache
```

Comment:
Sorry! It looks like I gave you bad advice here. Even though the sans-paretheses version works on Python 3.8, it apparently [doesn't](https://github.com/nchammas/flintrock/pull/342/checks?check_run_id=2525146078) in Python 3.6 or 3.7.

I'll fix it.